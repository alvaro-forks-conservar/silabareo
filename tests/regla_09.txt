abrupto
áptero
adopción
cápsula
amígdala
zigzag
astigmatismo
agnación
afgano
difteria
kafkiano
dip-ne-o