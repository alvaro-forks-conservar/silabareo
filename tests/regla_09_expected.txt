a-brup-to
áp-te-ro
a-dop-ción
cáp-su-la
a-míg-da-la
zig-zag
as-tig-ma-tis-mo
ag-na-ción
af-ga-no
dif-te-ria
kaf-kia-no
dip-ne-o